std_msgs/ColorRGBA color
---
string classified_color